server/api/store.go

```diff
@@ -610,6 +610,47 @@ func (h *storesHandler) GetStoreLimitScene(w http.ResponseWriter, r *http.Reques
 	h.rd.JSON(w, http.StatusOK, scene)
 }
 
+// StoreProgress contains status about a progress.
+type StoreProgress struct {
+	StoreID      uint64  `json:"store_id"`
+	Action       string  `json:"action"`
+	Progress     float64 `json:"progress"`
+	CurrentSpeed float64 `json:"current_speed"`
+	LeftSeconds  float64 `json:"left_seconds"`
+}
+
+// @Tags stores
+// @Summary Get store progress in the cluster.
+// @Produce json
+// @Success 200 {object} StoreProgress
+// @Failure 400 {string} string "The input is invalid."
+// @Failure 500 {string} string "PD server failed to proceed the request."
+// @Router /stores/{id}/progress [get]
```

Comment:
Can it be?
```
/stores/progress/{id}
/stores/progress/{action}
```

Reply:
Of cause, we can, but I'm not sure which one is better in API v1.